### ISSUE TITLE
Reviews blocks: rating has role="img" but missing aria-label

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1675]
Ratings shown in the reviews blocks are not accessible. They have `role="img"` but are missing an `aria-label` prop to define them:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/base/components/review-list-item/index.js#L126

Instead of directly solving it there, I think it would make more sense to reuse the logic from [`atomic/components/product/rating/index.js`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/atomic/components/product/rating/index.js), which was made accessible in #1656 and has exactly the same markup (only class names differ).

**Comment by mikejolley** [comment]
The markup is slightly different so we can't straight out swap the component but we can add the tag. Code has moved to here since the issue was created: 

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/f3b2b53435ffbc9ac49215846bad08ce74c7124b/assets/js/base/components/reviews/review-list-item/index.js#L122-L146
